### PR TITLE
azuredatastudio 1.44.1 > 1.47

### PR DESCRIPTION
**Pull request by Lillian-Violet**
## Description of changes

Update version for azuredatastudio

## Things done

checked if it compiles and runs and can do the basic functions like install extensions and connect to a database.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/misc/azuredatastudio/default.nix [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
+  libuuid,
+  libxkbcommon,
+  mesa,
+  nspr,
+  nss,
+  openssl,
+  pango,
+  systemd,
+  xorg,
+  zlib,
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/azuredatastudio/default.nix:32]: Please revert the formatting changes

File: pkgs/applications/misc/azuredatastudio/default.nix [hunk 2]
```diff
@@ -63,135 +62,134 @@ let
...
 
-  pname = "azuredatastudio";
-  version = "1.44.1";
+    desktopItems = [desktopItem urlHandlerDesktopItem];
 
-  desktopItems = [ desktopItem urlHandlerDesktopItem ];
+    src = fetchurl {
+      name = "${pname}-${version}.tar.gz";
+      url = "https://azuredatastudio-update.azurewebsites.net/${version}/linux-x64/stable";
+      sha256 = "sha256-DglBef+4g9Daj9vpchoyQCMDv2j0EURfLdhzEvw8Jsc=";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/azuredatastudio/default.nix:80]:
```suggestion
    hash = "sha256-DglBef+4g9Daj9vpchoyQCMDv2j0EURfLdhzEvw8Jsc=";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) when naming your commits.

**Comment by Lillian-Violet** [comment]
> Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) when naming your commits.

I squashed the commits in https://github.com/NixOS/nixpkgs/pull/269635/commits/2537d8b8f124995e503eb72cf8c4a7264bd1bdf2 and gave them a more descriptive name. I then removed the formatting changes, didn't realise I had my formatter on for that file.

**Pull request closed without merging**